redbot/cogs/trivia/data/lists/hockey.yaml

```
@@ -0,0 +1,2747 @@
+- Toronto St. Patricks
+- Toronto St. Pats
+- Toronto
+- St. Patricks
+- St. Pats
+# Hockey Trivia by aschwan41#0699
+What team tried to kick the Stanley Cup across a canal?:
+- The Ottawa Senators
+- Ottawa
+- Senators
+Who was drafted 133rd overall in the 1994 NHL Entry Draft?:
+- Daniel Alfredsson
+- Alfredsson
+What team hosted the 1999 NHL Entry Draft?:
+- Boston Bruins
+- Boston
+- Bruins
+What team hosted the 2000 NHL Entry Draft?:
+- Calgary Flames
+- Calgary
+- Flames
+What team hosted the 2001 NHL Entry Draft?:
+- Florida Panthers
+- Florida
+- Panthers
+What team hosted the 2002 NHL Entry Draft?:
+- Toronto Maple Leafs
+- Toronto
+- Maple Leafs
+- Leafs
+What team hosted the 2003 NHL Entry Draft?:
+- Nashville Predators
+- Nashville
+- Predators
+What team hosted the 2004 NHL Entry Draft?:
+- Carolina Hurricanes
+- Carolina
+- Hurricanes
+What team hosted the 2005 NHL Entry Draft?:
+- Ottawa Senators
+- Ottawa
+- Senators
+What team hosted the 2006 NHL Entry Draft?:
+- Vancouver Canucks
+- Vancouver
+- Canucks
+What team hosted the 2007 NHL Entry Draft?:
+- Columbus Blue Jackets
+- Columbus
+- Blue Jackets
+What team hosted the 2008 NHL Entry Draft?:
+- Ottawa Senators
+- Ottawa
+- Senators
+What team hosted the 2009 NHL Entry Draft?:
+- Montreal Canadiens
+- Montreal
+- Canadiens
+What team hosted the 2010 NHL Entry Draft?:
+- Los Angeles Kings
+- LA Kings
+- Kings
+- Los Angeles
+What team hosted the 2011 NHL Entry Draft?:
+- Minnesota Wild
+- Minnesota
+- Wild
+What team hosted the 2012 NHL Entry Draft?:
+- Pittsburgh Penguins
+- Pittsburgh
+- Penguins
+What team hosted the 2013 NHL Entry Draft?:
+- New Jersey Devils
+- New Jersey
+- Devils
+What team hosted the 2014 NHL Entry Draft?:
+- Philadelphia Flyers
+- Philadelphia
+- Flyers
+What team hosted the 2015 NHL Entry Draft?:
+- Florida Panthers
+- Florida
+- Panthers
+What team hosted the 2016 NHL Entry Draft?:
+- Buffalo Sabres
+- Buffalo
+- Sabres
+What team hosted the 2017 NHL Entry Draft?:
+- Chicago Blackhawks
+- Chicago Black Hawks
+- Blackhawks
+- Black Hawks
+- Chicago
+What team hosted the 2018 NHL Entry Draft?:
+- Dallas Stars
+- Dallas
+- Stars
+What team hosted the 2019 NHL Entry Draft?:
+- Vancouver Canucks
+- Vancouver
+- Canucks
+Who was the first player in NHL history to score 100 points in a single season, when he scored 126 total points?:
+- Phil Esposito
+- Esposito
+Who had their Detroit Red Wing number 12 retired in 1995?:
+- Sidney Abel
+- Sid Abel
+- Abel
+Which player was the first in NHL to score a full-strength, power play, short-handed, penalty shot, and open net goal in the same game?:
+- Mario Lemieux
+- Lemieux
+Which NHL player has played in the most regular season wins during his career?:
+- Scott Stevens
+- Stevens
+Which player was the first to win more than two individual trophies in one season?:
+- Stan Mikita won the Art Ross trophy, Lady Bing trophy, and Hart trophy two years in a row with the Chicago Blackhawks in 1966-67 and 1967-68.
```

```suggestion
- Stan Mikita won the Art Ross trophy, Lady Byng trophy, and Hart trophy two years in a row with the Chicago Blackhawks in 1966-67 and 1967-68.
```
Although this answer is *highly* specific, is it just for the auto-response when you fail?

---

Yes.